lib/digsync_web/live/create_group_live.ex

```diff
@@ -0,0 +1,37 @@
+defmodule DigsyncWeb.CreateGroupLive do
+  use DigsyncWeb, :live_view
+  alias Digsync.Accounts.Group
+  alias Digsync.Accounts
+
+  def mount(%{"user_id" => user_id}, _session, socket) do
+    current_user = socket.assigns.current_user
+    form = generate_form(current_user)
+    {:ok, assign(socket, actor: current_user, user_id: user_id, form: form)}
+  end
+
+  def generate_form(actor) do
```

Comment:
defp is probably better for local functions like this i'd also move them towards the bottom